scripts/globals/spells/spell_damage.lua

```diff
@@ -728,29 +732,30 @@ xi.spells.spell_damage.useDamageSpell = function(caster, target, spell)
 
     -- Get Tabled Variables.
-    local spellId      = spell:getID()
-    local skillType    = spell:getSkillType()
-    local spellElement = spell:getElement()
-    local statDiff     = caster:getStat(spellTable[spellId][stat]) - target:getStat(spellTable[spellId][stat])
+    local spellId         = spell:getID()
+    local skillType       = spell:getSkillType()
+    local spellElement    = spell:getElement()
+    local statDiff        = caster:getStat(spellTable[spellId][stat]) - target:getStat(spellTable[spellId][stat])
+    local spellDamageType = xi.damageType.ELEMENTAL + spellElement
 
     -- Variables/steps to calculate finalDamage.
-    local spellDamage                 = xi.spells.spell_damage.calculateBaseDamage(caster, target, spell, spellId, skillType, statDiff)
-    local multipleTargetReduction     = xi.spells.spell_damage.calculateMTDR(caster, target, spell)
-    local eleStaffBonus               = xi.spells.spell_damage.calculateEleStaffBonus(caster, spell, spellElement)
-    local magianAffinity              = xi.spells.spell_damage.calculateMagianAffinity(caster, spell)
-    local sdt                         = xi.spells.spell_damage.calculateSDT(caster, target, spell, spellElement)
-    local resist                      = xi.spells.spell_damage.calculateResist(caster, target,  spell, skillType, spellElement, statDiff, 0)
-    local magicBurst                  = xi.spells.spell_damage.calculateIfMagicBurst(caster, target,  spell, spellElement)
-    local magicBurstBonus             = xi.spells.spell_damage.calculateIfMagicBurstBonus(caster, target, spell, spellId, spellElement)
-    local dayAndWeather               = xi.spells.spell_damage.calculateDayAndWeather(caster, target, spell, spellId, spellElement)
-    local magicBonusDiff              = xi.spells.spell_damage.calculateMagicBonusDiff(caster, target, spell, spellId, skillType, spellElement)
-    local targetMagicDamageAdjustment = xi.spells.spell_damage.calculateTMDA(caster, target, spell)
-    local ebullienceMultiplier        = xi.spells.spell_damage.calculateEbullienceMultiplier(caster, target, spell)
-    local skillTypeMultiplier         = xi.spells.spell_damage.calculateSkillTypeMultiplier(caster, target, spell, skillType)
-    local ninSkillBonus               = xi.spells.spell_damage.calculateNinSkillBonus(caster, target, spell, spellId, skillType)
-    local ninFutaeBonus               = xi.spells.spell_damage.calculateNinFutaeBonus(caster, target, spell, skillType)
-    local undeadDivinePenalty         = xi.spells.spell_damage.calculateUndeadDivinePenalty(caster, target, spell, skillType)
-    local nukeAbsorbOrNullify         = xi.spells.spell_damage.calculateNukeAbsorbOrNullify(caster, target, spell, spellElement)
+    local spellDamage          = xi.spells.spell_damage.calculateBaseDamage(caster, target, spell, spellId, skillType, statDiff)
+    local MTDR                 = xi.spells.spell_damage.calculateMTDR(caster, target, spell)
```

Comment:
P.S. I don't know why this and others with similar changes are different here. Perhaps this got changed in a PR since my branch with changes was originally created.

Reply:
Turns out when I merged my changes here conflicted with the new style change applied in CI and I didn't know. Thank you CI.